addons/l10n_ec/data/template/account.tax-ec.csv

```diff
@@ -1,349 +1,349 @@
-"tax_withhold_vat_30","VAT withholding 30%","purchase","percent","50","-30.0","","VAT withholding 30%","","725","tax_group_withhold_vat","","base","invoice","","","","Retencion iva 30%"
+"tax_withhold_vat_30","30% WTH","purchase","percent","50","-30.0","VAT withholding 30%","-30%","","725","tax_group_withhold_vat","","base","invoice","","","","Retencion iva 30%"
 "","","","","","","","","","","","","tax","invoice","+725 (Reporte 104)","ec_vat_withhold_30","12",""
 "","","","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","","","tax","refund","-725 (Reporte 104)","ec_vat_withhold_30","12",""
-"tax_withhold_vat_50","VAT withholding 50%","purchase","percent","50","-50.0","","VAT withholding 50%","","727","tax_group_withhold_vat","","base","invoice","","","","Retencion iva 50%"
+"tax_withhold_vat_50","50% WTH","purchase","percent","50","-50.0","VAT withholding 50%","-50%","","727","tax_group_withhold_vat","","base","invoice","","","","Retencion iva 50%"
 "","","","","","","","","","","","","tax","invoice","+727 (Reporte 104)","ec_vat_withhold_50","12",""
 "","","","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","","","tax","refund","-727 (Reporte 104)","ec_vat_withhold_50","12",""
-"tax_withhold_vat_70","VAT withholding 70%","purchase","percent","50","-70.0","","VAT withholding 70%","","729","tax_group_withhold_vat","","base","invoice","","","","Retencion iva 70%"
+"tax_withhold_vat_70","70% WTH","purchase","percent","50","-70.0","VAT withholding 70%","-70%","","729","tax_group_withhold_vat","","base","invoice","","","","Retencion iva 70%"
 "","","","","","","","","","","","","tax","invoice","+729 (Reporte 104)","ec_vat_withhold_70","12",""
 "","","","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","","","tax","refund","-729 (Reporte 104)","ec_vat_withhold_70","12",""
-"tax_withhold_vat_100","VAT withholding 100%","purchase","percent","50","-100.0","","VAT withholding 100%","","731","tax_group_withhold_vat","","base","invoice","","","","Retencion iva 100%"
+"tax_withhold_vat_100","100% WTH","purchase","percent","50","-100.0","VAT withholding 100%","-100%","","731","tax_group_withhold_vat","","base","invoice","","","","Retencion iva 100%"
 "","","","","","","","","","","","","tax","invoice","+731 (Reporte 104)","ec_vat_withhold_100","12",""
 "","","","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","","","tax","refund","-731 (Reporte 104)","ec_vat_withhold_100","12",""
-"tax_sale_withhold_vat_10","VAT withholding 10%","sale","percent","60","-10.0","","VAT withholding 10%","","609","tax_group_withhold_vat","","base","invoice","","","","Retencion iva 10%"
+"tax_sale_withhold_vat_10","10% WTH","sale","percent","60","-10.0","VAT withholding 10%","-10%","","609","tax_group_withhold_vat","","base","invoice","","","","Retencion iva 10%"

```

Comment:
Although for the regular sales vat this shorter name seems like an improvement, I think it is not the case for withholds, in withholds we must differentiate from a "VAT" Withhold from a "Profit" Withhold.